test/index.js

```diff
@@ -125,11 +125,29 @@ describe('redux-mock-store', () => {
   it('subscribes to dispatched actions', (done) => {
     const store = mockStore();
     const action = { type: 'ADD_ITEM' };
-    
+
     store.subscribe(() => {
       expect(store.getActions()[0]).toEqual(action);
       done();
     });
     store.dispatch(action);
   });
+
+  it('can unsubscribe subscribers', function (done) {
```

Comment:
It's not possible to change this, as it will prevent setting the timeout for the asynchonous test, which is critical for it to work.

Arrow functions don't allow `this` to bound, so that particular functionality of the testing framework doesn't work with them.